infra/main.bicep

```diff
@@ -296,6 +302,17 @@ module searchServiceRoleOpenai 'core/security/role.bicep' = if (authType == 'rba
   }
 }
 
+// Storage Blob Data Reader
+module blobDataReaderRoleSearch 'core/security/role.bicep' = if (authType == 'rbac') {
```

Comment:
I think we need `Cognitive Services OpenAI User` as well, to generate embeddings

Reply:
It is already added https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/44bd0be460f498285daadf9ec2e9e0fc8b0f1dc5/infra/main.bicep#L835C1-L835C8

Reply:
That's the user role, we also need to give permissions to the Search Service